DiyorMarket.MVC/Lesson11/Views/Products/Index.cshtml

```diff
@@ -1,136 +1,130 @@
-﻿@model Lesson11.Models.Customer
+﻿@model Lesson11.Models.Product
 
 @{
     ViewData["Title"] = "Index";
     Layout = "~/Views/Shared/_DataGridLayout.cshtml";
 }
 <!-- Navigation link -->
-<nav aria-label="breadcrumb" style="margin-left: 20px;">
+<nav aria-label="breadcrumb">
     <ol class="breadcrumb">
         <li class="breadcrumb-item"><a href="/dashboard/index"><i class="fa fa-home" style="color: white"></i></a></li>
         <li class="breadcrumb-item active" aria-current="page">Products List</li>
     </ol>
 </nav>
-    <form asp-controller="Products" asp-action="Index" class="mb-2">
-        <input type="hidden" name="prevCategoryId" value="@ViewBag.SelectedCategory?.Id" />
-        <div class="container" style="margin-top: 50px; margin-left: 15px;">
-            <div class="row row-cols-sm-1 align-items-center">
-                <div class="d-flex flex-row justify-content-center gap-4">
-                    <!-- Search -->
-                        <div class="col-md-3 mb-4">
-                            <div class="input-group">
-                                <input type="hidden" name="prevCategoryId" value="@ViewBag.SelectedCategory?.Id" />
-                                <input type="text" class="form-control" placeholder="Search product..." name="searchString" value="@ViewBag.SearchString">
-                                <div class="input-group-append">
-                                    <button type="submit" class="btn btn-primary">
-                                        <i class="fa fa-search"></i> Search
-                                    </button>
-                                </div>
-                            </div>
-                        </div>      
-                    <!-- Filter -->
-                    <div class="col-md-3">
-                        <div class="input-group">
-                            <select class="form-control" name="categoryId">
-                                <option value="" selected disabled>@ViewBag.SelectedCategory.Name</option>
-                                @foreach (var category in ViewBag.Categories)
-                                {
-                                    <option value="@category.Id" selected="@(category.Id == ViewBag.categoryId)">
-                                        @category.Name
-                                    </option>
-                                }
-                            </select>
-                        </div>
+<form asp-controller="Products" asp-action="Index">
+    <input type="hidden" name="prevCategoryId" value="@ViewBag.SelectedCategory?.Id" />
```

Comment:
we can give full descriptive name such as **previousCategoryId** or **previousCategory**